firmware/src/usb_descriptors.c

```diff
@@ -143,7 +144,7 @@ static const uint8_t g_config_descriptor[] = {
   USB_DESCRIPTOR_ENDPOINT,  // Endpoint Descriptor
   0x2 | USB_EP_DIRECTION_IN,  // EndpointAddress ( EP2 IN INTERRUPT)
   USB_TRANSFER_TYPE_INTERRUPT,  // Attributes type of EP (INTERRUPT)
-  0x0A, 0x00,  // Max packet size of this EP
+  0x10, 0x00,  // Max packet size of this EP
```

Comment:
Yup, 0x10 is 16 bytes which is one of the standard sizes. I'm not sure why the original plugin generated a max size of 10 bytes, that seems wrong :).